src/sentry/features/__init__.py

```diff
@@ -73,6 +73,7 @@
 default_manager.add("organizations:discover-events-rate-limit", OrganizationFeature, True)
 default_manager.add("organizations:discover-query-builder-as-landing-page", OrganizationFeature, True)
 default_manager.add("organizations:ds-prioritise-by-project-bias", OrganizationFeature)
+default_manager.add("organizations:ds-prioritise-by-transaction-bias", OrganizationFeature)
```

Comment:
will this be controlled by flagr or AM2 plan?  if AM2 plan, then we are good, otherwise a `True` is needed here

Reply:
no, it's only for us to hide it until we release it. So it should be controlled by UI toggle and enabled for all clients.